voxblox/include/voxblox/core/layer.h

```diff
@@ -225,9 +225,14 @@ class Layer {
   bool isCompatible(const LayerProto& layer_proto) const;
   bool isCompatible(const BlockProto& layer_proto) const;
   bool saveToFile(const std::string& file_path) const;
+  bool saveToFile(const std::string& file_path, bool clear_file) const;
```

Comment:
do you need two functions, can't you just use a default value for clear_file?

Reply:
Ah shit I remembered google style guide strongly discouraging default arguments, but checked it again and I'm wrong. Yeah here it makes waaaaaay more sense to do that. Changing.